hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/XceiverServerRatis.java

```diff
@@ -583,7 +585,8 @@ public void submitRequest(ContainerCommandRequestProto request,
       try {
         reply = server.submitClientRequestAsync(raftClientRequest)
             .get(requestTimeout, TimeUnit.MILLISECONDS);
-      } catch (Exception e) {
+      } catch (InterruptedException | ExecutionException | TimeoutException e) {
+        Thread.currentThread().interrupt();
```

Comment:
I think this can be handled in a graceful way.

Reply:
Thanks to @cku328 's review.
Thanks for reminding.